packages/ts/src/containers/single-container/index.ts

```diff
@@ -57,6 +57,9 @@ export class SingleContainer<Data> extends ContainerCore {
       tooltip.setComponents([this.component])
     }
 
+    // Defs
+    this.element.appendChild(this._svgDefs.node())
```

Comment:
Won't this append the defs over and over again on every `updateContainer` call?

Reply:
No, because `removeAllChildren` is called initially (on [line 48](https://github.com/f5/unovis/blob/48662c5466d7ff09b1546ddabb9c519389c9c9f8/packages/ts/src/containers/single-container/index.ts#L48C5-L48C30))